packages/trade/exchange-core/src/MatchingEngine.ts

```diff
@@ -327,11 +327,11 @@ export class MatchingEngine<TProduct, TProductFilter> {
         const hasPriceMatched = ask.price <= bid.price;
         const sameOwner = bid.userId === ask.userId;
 
-        this.logger.debug(
-            `[ask: ${ask.id} <-> bid: ${
-                bid.id
-            }] hasProductMatched: ${hasProductMatched} hasAskVolume: ${hasAskVolume} hasBidVolume: ${hasBidVolume} hasPriceMatched: ${hasPriceMatched} notSameOwner: ${!sameOwner}`
-        );
+        // this.logger.debug(
+        //     `[ask: ${ask.id} <-> bid: ${
+        //         bid.id
+        //     }] hasProductMatched: ${hasProductMatched} hasAskVolume: ${hasAskVolume} hasBidVolume: ${hasBidVolume} hasPriceMatched: ${hasPriceMatched} notSameOwner: ${!sameOwner}`
+        // );
```

Comment:
Any reason these should be left commented?

Reply:
I'm trying to understand test logs because e2e tests fail from time to time. I'll return these lines after I understand the reason